src/server/saved_objects/service/lib/repository.js

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 
+import { get, snakeCase } from 'lodash';
```

Comment:
I'm concerned about importing from lodash. I think we're moving away from lodash. @spalger thoughts?
This library is also undergoing a move to the new platform. We talked about locking down the saved objects client to changes while that is happening. What do you think @epixa?